src/backing/client/crypto_key_version_algorithm_test.cc

```diff
@@ -83,6 +83,30 @@ TEST_P(CryptoKeyVersionAlgorithmTest, UnderlyingValueMatchesProtoValues) {
   EXPECT_EQ(CryptoKeyVersionAlgorithmToInt(mapping.actual), mapping.proto);
 }
 
+TEST(CryptoKeyVersionAlgorithmTest, CryptoKeyVersionAlgorithmToString) {
```

Comment:
One possibility is to use a TEST_P and cover all the cases here.

I'm not sure about whether it truly is necessary to do that, though. But I think it could be nice to at least add a check to verify that the mapping is **1 to 1 and onto**. For instance, in a for loop you could make a dictionary in the unit tests and check that no answer string is duplicate, and that the number of unique keys is equal to the number of unique values?

That'd be able to catch copy & paste errors where an engineer forgets to update the answer string or the enum value.

Reply:
Ok, I'll do that. I was trying to avoid just enumerating the strings that are already in the code, so this sounds like a good plan.

Reply:
Handling in #75